1-JavaScript/Week2/homeworkSolition/ex1-sol-giveCompliment.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+function giveCompliment(name) {
+  // TODO complete this function
+  let compliments = [
+    "bravo",
+    "great",
+    "nice",
+    "excellent",
+    "amazing",
+    "better",
+    "wonderful",
+    "verygood",
+    "Super",
+    "fantastoc",
+  ];
+  let randomCompliment = Math.floor(Math.random() * compliments.length);
+  //   it is only possible to return 2 or more values using array using [] or as object in {}
+  //   return [compliments[randomCompliment], name];
+  let displayComplimetn = `You are ${compliments[randomCompliment]} ${name} !`;
+  return displayComplimetn;
```

Comment:
Nice job writing this function 💪 One small tip: you can directly return the string instead of storing it in a separate variable 🙂 

```suggestion
  return `You are ${compliments[randomCompliment]} ${name} !`;
```